chart/k8gb/templates/external-dns/rbac.yaml

```diff
@@ -29,7 +29,7 @@ kind: ServiceAccount
 metadata:
   name: k8gb-external-dns
   namespace: {{ .Release.Namespace }}
-{{ if .Values.route53.enabled }}
+{{ if and .Values.route53.enabled .Values.route53.irsaRole }}
```

Comment:
If we do this oposite? 
```{{ if .Values.route53.irsaRole }} ```

Reply:
how is it the opposite?

Reply:
Well irsaRole asumes you have route53 is enabled

Reply:
that's exactly the logic I want to convey. if route53 enabled AND irsaRole is specified then inject the annotation. Do you see any problem with this logic?

Reply:
It is duplicit and hard to read

Reply:
but also not a rocket science and more reliable

Reply:
Also it enables us to keep example at https://github.com/k8gb-io/k8gb/blob/master/chart/k8gb/values.yaml#L68 and not forcing it to `false`.

Reply:
@k0da also added inline clarification comment please re-review